regression/smt2_strings/.gitignore

```diff
@@ -0,0 +1 @@
+!*.smt2
```

Comment:
That would mean that Makefile, CMakeLists.txt, *.desc are also ignore, which doesn't seem right.

Reply:
I don't understand. I need to discard the ignore that is introduced here:
https://github.com/diffblue/cbmc/blob/develop/.gitignore#L59
and which is useful on the regression tests where cbmc is run with the `--smt2` option.

Reply:
Oops, I wasn't aware of that interaction.

Reply:
I don't think that cbmc still writes the file on disk when run with --smt2?